packages/components/src/avatar/avatar.tsx

```diff
@@ -51,23 +51,23 @@ type ChannelAvatarProps = {
 
 type CommunityAvatarProps = {
   type: 'community'
-  size: 80 | 32 | 28 | 24 | 20
+  size: 80 | 32 | 28 | 24 | 20 | 16
   name: string
   src?: string
   backgroundColor?: ColorTokens
 }
 
 type AccountAvatarProps = {
   type: 'account'
-  size: 80 | 48 | 32 | 28 | 24 | 20
+  size: 80 | 48 | 32 | 28 | 24 | 20 | 16
```

Comment:
> This looks like an Account Avatar size 16, but https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n/Design-System-for-Desktop%2FWeb?type=design&node-id=483-19407&mode=design&t=Qsa7IQmhebwM2PVY-4 does not have it defined. –  https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n?node-id=1336:34320&mode=design#541620575

Reply:
Avatar size 16 added in https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n/Design-System-for-Desktop%2FWeb?type=design&node-id=18400-10276&mode=design&t=gYqNYTsPL7i7Np3U-4.